core/modules/server/server-sent-events.js

```diff
@@ -0,0 +1,74 @@
+/*\
+title: $:/core/modules/server/server-sent-events.js
+type: application/javascript
+module-type: library
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/**
+ * @param {string} prefix
```

Comment:
The core doesn't use JSDoc, just plain text for code comments

Reply:
I use it to give VS Code the proper intellisense. TiddlyWiki is really hard to work with otherwise.

Reply:
Yes, for us familiar with type script, interface with typing is very nice.